server/index.js

```diff
@@ -33,6 +34,9 @@ const cache = initCache(config.server);
 const app = express();
 const port = argv.port || config.server.port;
 
+// gzip
+app.use(compression());
```

Comment:
This needs to be gated behind a config that is only set to `true` when starting in local mode.

When running on actual servers gzip is activated at a different layer.